website/siteConfig.js

```diff
@@ -40,6 +40,11 @@ const users = [
     image: 'img/vonage-logo.png',
     infoLink: 'https://www.vonage.com/',
   },
+  {
+    caption: 'FactorialHR',
+    image: 'img/factorial-logo.png',
+    infoLink: 'https://www.factorialhr.com/',
+  },
```

Comment:
Could you move this entry to right after `Chan Zuckerberg Initiative` (line 28)?

Reply:
sure!

Reply:
done :)